qt/widgets/common/inc/MantidQtWidgets/Common/DataProcessorUI/WhiteList.h

```diff
@@ -37,30 +44,34 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 File change history is stored at: <https://github.com/mantidproject/mantid>.
 Code Documentation is available at: <http://doxygen.mantidproject.org>
 */
+
 class EXPORT_OPT_MANTIDQT_COMMON WhiteList {
 public:
-  WhiteList() : m_lastIndex(0){};
+  using const_iterator = ConstColumnIterator;
   virtual ~WhiteList(){};
```

Comment:
Is this necessary? It will suppress the default move constructor/assignment operator.